src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -106,12 +109,23 @@ const AttendanceFormBuilder = () => {
 
   const onSubmit = async (values: AttendanceFormValues) => {
     try {
-      const result = await createAttendance(values);
-      if (!result) {
-        toast({
-          title: "Attendance submit successfully",
-          description: "You will receive message shortly",
-        });
+      if (paramId) {
+        const updatedResult = await updateAttendance(Number(paramId), values);
```

Comment:
not a good approach pass only one prop and then convert id into number on server component

Reply:
Fixed